lambdas/select-cri/src/main/java/uk/gov/di/ipv/core/selectcri/SelectCriHandler.java

```diff
@@ -144,7 +144,14 @@ private JourneyResponse getNextWebJourneyCri(
                         passportCriId,
                         passportCriId,
                         userId);
-        if (passportResponse.isPresent()) {
+        Optional<JourneyResponse> drivingLicenceResponse =
+                getCriResponse(
+                        visitedCredentialIssuers,
+                        currentVcStatuses,
+                        drivingLicenceCriId,
+                        drivingLicenceCriId,
+                        userId);
+        if (passportResponse.isPresent() && drivingLicenceResponse.isPresent()) {
```

Comment:
Just out of interest, does this still work if the logic is:

``` if (passportResponse.isPresent() && drivingLicenceResponse.isPresent())```

Reply:
🤔

Reply:
It wouldn’t as response.isPresent() is true if that CRI hasn’t been visited yet - so we need both to not be present

Reply:
Fair enough. Thank you.